pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -57,6 +58,8 @@ buildPythonPackage rec {
     #   url = "https://github.com/openai/triton/commit/fc7c0b0e437a191e421faa61494b2ff4870850f1.patch";
     #   hash = "sha256-f0shIqHJkVvuil2Yku7vuqWFn7VCRKFSFjYRlwx25ig=";
     # })
+  ] ++ lib.optionals (!cudaSupport) [
+    ./0000-dont-download-ptxas.patch
```

Comment:
Btw https://github.com/openai/triton/commit/cde42e62215cdb03dd150ac6cf0ace9a38605c1b got merged but the patch probably won't apply at the current version, so maybe we handle this in later PRs